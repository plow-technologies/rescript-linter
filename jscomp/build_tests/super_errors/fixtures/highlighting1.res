let a: int = "hel

lo"
