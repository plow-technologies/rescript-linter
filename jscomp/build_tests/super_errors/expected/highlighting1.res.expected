
  [1;31mWe've found a bug for you![0m
  [36m/.../fixtures/highlighting1.res[0m:[2m1:14-3:3[0m

  [1;31m1[0m [2mâ”‚[0m let a: int = [1;31m"hel[0m
  [1;31m2[0m [2mâ”‚[0m 
  [1;31m3[0m [2mâ”‚[0m [1;31mlo"[0m
  4 [2mâ”‚[0m 

  This has type: [1;31mstring[0m
  But it's expected to have type: [1;33mint[0m
  
  You can convert [1;33mstring[0m to [1;33mint[0m with [1;33mBelt.Int.fromString[0m.